tests/coverage.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -eux
+
+set -o pipefail
+
+# Script Collection CI
+# Once working, we may move this script elsewhere - gundalow
+
+
+if find tests/output/coverage/ -mindepth 1 -name '.*' -prune -o -print -quit | grep -q .; then
+    stub=""
+
+    # shellcheck disable=SC2086
+    ansible-test coverage xml -v --requirements --group-by command --group-by version ${stub:+"$stub"}
+
+    # upload coverage report to codecov.io
+    for file in tests/output/coverage/coverage=*.xml; do
+        flags="${file##*/coverage=}"
+        flags="${flags%.xml}"
+        flags="${flags//=/,}"
+        flags="${flags//[^a-zA-Z0-9_,]/_}"
+
+        bash <(curl -s https://codecov.io/bash) \
+            -f "${file}" \
+            -F "${flags}" \
+            -t 83cd8957-dc76-488c-9ada-210dcea51633 \
```

Comment:
Arbitrary collections shouldn't be uploading code coverage for the ansible/ansible repo.

Reply:
Ah, I thought I had corrected the token, thank you.